app/src/main/java/ca/deflector/passable_shots/MainActivity.java

```diff
@@ -44,7 +45,20 @@ public void onPictureTaken(byte[] data, Camera camera) {
                 // call encryption here
                 try {
                     byte[] encrypted = EncryptionSystem.encrypt(data);
-                    FileOutputStream fos = new FileOutputStream(Environment.getExternalStorageDirectory() + "/passable." + System.currentTimeMillis());
+                    // determine latest number
+                    int maxPassableFile = 0;
+                    for (File f : Environment.getExternalStorageDirectory().listFiles())
```

Comment:
I think we can avoid inadvertent bugs in the future by putting the contents of `for` inside `{}`. But if you prefer it this way, I don't have a strong objection. Go ahead and merge as you see fit.